src/gentropy/colocalisation.py

```diff
@@ -27,20 +31,52 @@ def __init__(
             credible_set_path (str): Input credible sets path.
             study_index_path (str): Input study index path.
             coloc_path (str): Output Colocalisation path.
+            colocalisation_method (ColocalisationMethod): Colocalisation method. Available methods are: ECAVIAR, COLOC.
         """
         # Extract
-        credible_set = StudyLocus.from_parquet(
-            session, credible_set_path, recursiveFileLookup=True
+        credible_set = (
+            StudyLocus.from_parquet(
```

Comment:
It works for now but this paragraph is problematic. "Susie" and "coloc" are hardcoded.

Reply:
"coloc" is no longer hardcoded.
I don't see how i can not hardcode "SuSie" without plaguing the code with parameters.

Reply:
I think what @d0choa meant is that the applied colocalization method should know what requirements the data should accomodate not the step. Eg. each coloc methods would have a set of criteria that would apply on the dataset, in that sense it would be the method's reponsability to enforce that the data is right.

Reply:
Apologies I had missed this comment.
I think the responsibility is in the dependency injection step, not in the method.
In any case, `coloc` is already capturing that scenario because it's a method that **requires** having logBFs. If none of the credible sets have them, the data will pass through and the output coloc metrics will be 0 (this is theoretically true, I just found a bug).

Also, considering that the input of the colocalisation function is an Overlaps object, I think it makes sense to add the filter to just include SuSie credible sets at the step level because we won't extract overlaps for credible sets for which we can't get coloc results.